projects/vdk-plugins/vdk-lineage/README.md

```diff
@@ -0,0 +1,30 @@
+# VDK Lineage
+
+VDK Lineage plugin provides lineage data (input data -> job -> output data) information and send it to pre-configured
+destination.
+
+At POC level currently. It collect lineage information for each job run and for each executed query.
+Query execution is currently before it's executed (so not query status is logged).
```

Comment:
I'm confused as to what this sentence means. Maybe rephrase it?

Reply:
I re-wrote the README entirely . Hopefully it's now clearer/better.